app/models/version.rb

```diff
@@ -24,7 +24,7 @@ def self.get_team_id_from_item_type(item_type, item)
       item.annotation&.get_team&.last
     when 'Relationship'
       item.source&.project&.team_id
-    when 'ProjectMedia', 'ProjectSource'
+    when 'ProjectMedia'
```

Comment:
We will need a migration to remove `ProjectSource` entries from versions.

Reply:
yes here is the the list for the rake task migration

- Remove ProjectSource from PG
- Remove ProjectSource annotations
- Remove versions related to ProjectSource & ProjectSource annotation
- Remove cache that key start with `project_source_id_cache_for_project_media_`
- Remove ProjectSource items from ES

Reply:
great!